src/main/java/com/alex/eshop/keycloak/KeycloakService.java

```diff
@@ -25,4 +41,66 @@ public AccessTokenResponse getToken() {
         AuthzClient authzClient = AuthzClient.create(authzConf);
         return authzClient.obtainAccessToken();
     }
+
+    public RealmResource realmResource() {
+        return keycloakClientFactory.getInstance()
+                .realm(applicationProperties.getKeycloak().getRealm());
+    }
+
+    public UsersResource usersResource() {
+        return keycloakClientFactory.getInstance()
+                .realm(applicationProperties.getKeycloak().getRealm())
+                .users();
+    }
+
+    public List<RoleRepresentation> getRoleRepresentation() {
```

Comment:
this method can be used only in one case. not a lot of sense from it.
Additionally, I can say, that user will not only get role `ROLE_USER` but also other default role by keycloak.